lib/cogctl/actions/relay_groups/add.ex

```diff
@@ -2,27 +2,27 @@ defmodule Cogctl.Actions.RelayGroups.Add do
   use Cogctl.Action, "relay-groups add"
 
   def option_spec() do
-    [{:name, :undefined, :undefined, {:string, :undefined}, 'Relay Group name (required)'},
-     {:relay, :undefined, 'relay', {:string, :undefined}, 'Relay name (required)'}]
+    [{:relay_group, :undefined, :undefined, {:string, :undefined}, 'Relay Group name (required)'},
+     # Technically this will just be the first relay
+     {:relays, :undefined, :undefined, {:string, :undefined}, 'Relay names (required)'}]
   end
 
-  def run(options, _args, _config, %{token: nil}=endpoint) do
-    with_authentication(endpoint, &run(options, nil, nil, &1))
-  end
-  def run(options, _args, _config, endpoint) do
-    case convert_to_params(options, [relay: :required,
-                                     name: :required]) do
+  def run(options, args, _config, endpoint) do
+    # At least one relay is required, so we specify that here
+    case convert_to_params(options, [relay_group: :required,
+                                     relays: :required]) do
       {:ok, params} ->
-        do_add(endpoint, params)
+        params = %{params | relays: [params.relays | args]}
```

Comment:
Yeah, I saw that. The thing is the list type isn't supported by `getopt`. `getopt` just parses it as a string. We then manually split a string at a comma. So the input for that looks like this: `--members=member1,member2,member3`. If you deviate from that at all, by adding a space between entries for example, it crashes. That seemed more fragile.

In my implementation here the args are based on position.